float64 sx  # 步长
float64 sy   # 步宽
float64 var_theta  # 转角增量
bool walk_with_ball  # 是否带球行走/动态踢球
bool stop_walk   # 停止行走标志位